### PR TITLE
Ddcnl 3316 - Updated new API documentation

### PR DESCRIPTION
**Pull request by Dandyboyo2**

### CODE REVIEW

File: public/api/conf/1.0/testdata/get-pensions-dashboard.md [hunk 1]
```diff
@@ -0,0 +1,178 @@
+<table>
+    <thead>
+        <tr>
+            <th>Scenario</th>
+            <th>NINO</th>
+            <th>Response</th>
+        </tr>
+    </thead>
+    <tbody>
```
Comment (lewey) [inline public/api/conf/1.0/testdata/get-pensions-dashboard.md:9]: This tag does not get closed

File: public/api/conf/1.0/testdata/get-pensions-dashboard.md [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+            <td><p>Taxpayer who has the manual correspondence indicator flag</p></td>
+            <td><p><code>LG007101A</code></p></td>
+            <td><p>403 (Forbidden) {"code": "EXCLUSION_MANUAL_CORRESPONDENCE"}</p></td>
+        </tr>
+        <tr>
+            <td><p>Taxpayer who has a date of death</p></td>
+            <td><p><code>LG005801A</code></p></td>
+            <td><p>403 (Forbidden) {"code": "EXCLUSION_DEAD"}</p></td>
+        </tr>
+    </trs>
```
Comment (lewey) [inline public/api/conf/1.0/testdata/get-pensions-dashboard.md:177]: no matching tag for TRS

File: public/api/conf/1.0/testdata/get-pensions-dashboard.md [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+            <th>NINO</th>
+            <th>Response</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <th colspan="3">More than 10 qualifying years</th>
+        </tr>
+        <tr>
+            <th colspan="3">Forecast</th>
```
Comment (lewey) [inline public/api/conf/1.0/testdata/get-pensions-dashboard.md:14]: As this is test data are we happy to have in lined styling?

Comment (thirlz95) [reply public/api/conf/1.0/testdata/get-pensions-dashboard.md:14]: I'm happy with this using colspan, it's used plenty within other HMRC applications

File: public/api/conf/1.0/testdata/get-pensions-dashboard.md [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+            <td><p>Taxpayer who is predicted to receive less than the full rate</p></td>
+            <td><p><code>LG001101A</code></p></td>
+            <td><p>200 (OK) Payload as regular example above</p></td>
+        </tr>
+        <tr>
+            <th colspan="3">Personal maximum</th>
+        </tr>
+        <tr>
+            <td><p>Taxpayer who is predicted to receive the full rate by paying gaps</p></td>
+            <td><p><code>LG000601A</code></p></td>
```
Comment (thirlz95) [inline public/api/conf/1.0/testdata/get-pensions-dashboard.md:31]: I believe you'll be okay just using <td><code> LG000601A</code></td>

File: public/api/conf/1.0/testdata/get-pensions-dashboard.md (hunk 2 above)
Comment (thirlz95) [reply public/api/conf/1.0/testdata/get-pensions-dashboard.md:177]: I believe this should be the closing tag for </tbody>

Comment (thirlz95) [reply public/api/conf/1.0/testdata/get-pensions-dashboard.md:177]: tbody

File: public/api/conf/1.0/testdata/get-pensions-dashboard.md (hunk 4 above)
Comment (thirlz95) [reply public/api/conf/1.0/testdata/get-pensions-dashboard.md:31]: just remove the p tag and just have the code tag, github formatted the tags i've done

**Review by thirlz95 — APPROVED** [review]

File: public/api/conf/1.0/testdata/get-pensions-dashboard.md (hunk 4 above)
Comment (Dandyboyo2) [reply public/api/conf/1.0/testdata/get-pensions-dashboard.md:31]: remove the p tag on all instances within the file?

Comment (thirlz95) [reply public/api/conf/1.0/testdata/get-pensions-dashboard.md:31]: just where the <code> tags are used for the nino's

File: public/api/conf/1.0/testdata/get-pensions-dashboard.md (hunk 2 above)
Comment (Dandyboyo2) [reply public/api/conf/1.0/testdata/get-pensions-dashboard.md:177]: closed with </tbody> in place of </trs>

File: public/api/conf/1.0/testdata/get-pensions-dashboard.md (hunk 4 above)
Comment (Dandyboyo2) [reply public/api/conf/1.0/testdata/get-pensions-dashboard.md:31]: p tags removed around NINOs in this file and get-state-pension.md (where I cloned data for the new dashboard.md file)

File: public/api/conf/1.0/testdata/get-pensions-dashboard.md (hunk 1 above)
Comment (Dandyboyo2) [reply public/api/conf/1.0/testdata/get-pensions-dashboard.md:9]: closed with /tbody in place of /trs

**Pull request merged**